test/e2e/gameserver_test.go

```diff
@@ -259,7 +259,8 @@ func TestGameServerUnhealthyAfterDeletingPod(t *testing.T) {
 }
 
 func TestGameServerRestartBeforeReadyCrash(t *testing.T) {
-	t.Parallel()
+	// TODO(#2445): The feature is flaky when pod updates are slow, run serially to avoid.
```

Comment:
I couldn't find documentation on referencing issues from code, is there a syntax for it? Will this work, or will agones#2445 work?

Reply:
Well, this is clear enough. :)